Update marzban panel script and add gozargah node script for v1 compatibility and externalize their compose files

mahdim10 opened the pull request:
This pull request includes the following changes:

- Updated the install scripts to support Marzban v1.

- Added support for postgresql and timescaledb as valid database_type options (v1 only).

- Introduced the --pre-release flag to allow installation of beta or pre-release versions of Marzban Panel.

- The gozargah node script also supports the --pre-release flag, and the --version argument has been added to the script as well.

- Externalized Docker Compose files to improve maintainability and flexibility.

mahdim10 committed:
feat: update marzban install script for v1 compatibility and externalize compose files

mahdim10 committed:
feat: add gozargah-node setup script and compose file

ImMohammad20000 commented on marzban.sh:
```diff
@@ -708,237 +707,108 @@ update_core_command() {
+    fi
+    
+    FILES_URL_PREFIX="https://raw.githubusercontent.com/Gozargah/Marzban/"
+    COMPOSE_FILES_URL_PREFIX="https://raw.githubusercontent.com/Gozargah/Marzban-scripts/master"
```
changr repo name to this repo!

mahdim10 committed:
docs: update README for Marzban v1 and Gozargah-node

- Updated examples for compatibility with Marzban v1
- Added installation instructions for Gozargah-node
- Included examples and tips for using --name flag for both Gozargah-node and Marzban-Node
- Noted deprecation of Marzban-node

mahdim10 committed:
fix: update repo name to point to forked repository
